src/main/java/com/beantastic/api/models/entities/BeanClassStatistic.java

```diff
@@ -5,13 +5,15 @@
 
 @Data
 @Entity
-@Table
+@Table(name = "beanclassstatistic")
```

Comment:
I see that you made it all lowercase (table, column and joincolumn). For readability I would have made it camelCase - the db does't care. But it's honestly not enough of an issue to have to change.

Reply:
I think with the JPA it was causing weird issues if it wasnt all lowercase like it is in the DB